pubsub.go

```diff
@@ -333,8 +336,9 @@ func (p *PubSub) processLoop(ctx context.Context) {
 			}
 
 			delete(p.peers, pid)
-			for _, t := range p.topics {
-				delete(t, pid)
+			for t, tmap := range p.topics {
+				delete(tmap, pid)
+				p.notifySubscriberLeft(t, pid)
```

Comment:
this is going to emit phantom notifications, as it might not be in the map.

Reply:
oops, pretty silly mistake there. Thanks